docs/general/events.md

```diff
@@ -3,13 +3,35 @@ linkTitle: Events
 aliases: [docs/specs/semconv/general/events-general]
 --->
 
-# Semantic Conventions for Event Attributes
+# Semantic Conventions for Events
 
 **Status**: [Experimental][DocumentStatus]
 
-This document describes the attributes of standalone Events that are represented
+This document describes the characteristics of standalone Events that are represented
 in the data model by `LogRecord`s.
 
+Semantically, an Event is a named occurrence at an instant in time. It signals that
+"this thing happened at this time" and provides additional specifics about the occurrence.
+Examples of Events might include things like uncaught exceptions, button clicks, user logout,
+network connection severed, etc.
+
+In OpenTelemetry, Events are implemented as a specific type of `LogRecord` that conforms to
+the conventions included here, and Events
+[have their own API](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/event-api.md).
+The API abstracts away knowledge of `LogRecord` so that users only deal with Event
+semantics.
+
+In addition to a required name, an Event may contain a _payload_ of any type permitted by the
+[LogRecord body](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#field-body).
+In its implementation, the Event _payload_ will constitute the `Body` of the `LogRecord`.
+Like all other OpenTelemetry signals, an Event has optional attribute metadata that helps describe
+the event context.
+
+Over time, some Events will be specified by OpenTelemetry and will have documented payload structure,
+field semantics, and stability and requirement levels. Other events may be user-defined and carry
+bespoke user semantics. When an Event name exists in the semantic conventions, its _payload_
+structure and semantics will also be defined.
```

Comment:
I really want to see some specifics on how events can be described in semconv, specifically wrt payload/body.
It's not possible today and without defining a mechanism this PR seems incomplete.

Reply:
Thanks @lmolkova. This is definitely intended to be incremental progress and not a comprehensive solve for events. We (the events working group) are working through some of the ideas around semconv and how it might play with existing tooling (or not). 

This PR was not intending to do all of that in one shot, but rather simply solidify the decision that the event payload lives in the log record body. 

How event payloads are eventually described in semantic conventions, how schemas might be represented, how markdown might be generated from yaml/schema files, how changes to existing events can be expressed in schema version files....all of that is still being worked out.

Reply:
Thank you for the explanation! 

So if I understand correctly, this PR only declares that payload should be described by the semconv and there will be a way. 
I'm glad there is a concensus there, but without specifics, what's the value of having this in the semconv?

Reply:
Previously, there was an open question on where the event payload would live -- does it go in the body of the LogRecord, or does it go into an attribute with a specific name like `event.data`? The benefit is having this question answered and providing a foundation on which to build future semconv.

Reply:
I'm going to resolve it in favor of https://github.com/open-telemetry/semantic-conventions/issues/695